lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -532,10 +653,16 @@ func (c *ContainerProxy) Signal(vc *viccontainer.VicContainer, sig uint64) error
 			}
 		}
 
-		c.Wait(vc, -1*time.Second)
+		// Wait awhile to see if signal killed the container
+		if _, _, _, err := c.Wait(vc, killWaitBeforeForce); err != nil {
```

Comment:
OK that makes sense. Do we need a wait then? Why is an error from the PL in response to the signal not enough to just go straight to the stop() call?